utils-metrics/pom.xml

```diff
@@ -120,6 +120,7 @@
     <dependency>
       <groupId>org.bdgenomics.utils</groupId>
       <artifactId>utils-misc_2.10</artifactId>
+      <scope>compile</scope>
```

Comment:
Oh, fascinating. I'm an emacs user as well, so I can't comment on why IDEA was giving you the business.